cmd/system-probe/api/server.go

```diff
@@ -33,6 +36,22 @@ func StartServer(cfg *config.Config) error {
 		return fmt.Errorf("failed to create system probe: %s", err)
 	}
 
+	// Setup telemetry server
+	port := cfg.DefaultSystemProbeExpVarPort
+	if ddconfig.Datadog.GetBool("telemetry.enabled") {
+		http.Handle("/telemetry", telemetry.Handler())
+	}
+	expvarServer := &http.Server{
+		Addr:    "127.0.0.1:" + port,
```

Comment:
Since the port number is read from a config file, we may want to do some validation that it's a valid port number before attempting to use it